### DISCUSSION TITLE
Hockey NHL problème de cote PMU

### DISCUSSION THREAD

**Discussion by kreon76** [pretrehr/Sports-betting#16] [General]
Bonjour, je me permets encore de vous remonter un petit bug...
![python_2021-02-04_18-12-06](https://user-images.githubusercontent.com/78261069/106929472-afb7d780-6714-11eb-85cf-d88185f04d80.png)
Au lieu de me mettre une cote aux alentours des 1.4 il prend une cote à 4.4 pour Montreal sur PMU

Merci par avance

**Comment by kreon76** [comment]
![image](https://user-images.githubusercontent.com/78261069/106930649-14276680-6716-11eb-9f7b-57aaa274e529.png)
Alors que sur PMU la cote est bien de 1.4
J'ai lancé 2 scrap et c toujours pareil

**Reply by pretrehr** [reply]
C'est bon c'est corrigé, merci pour cette trouvaille !

**Comment by kreon76** [comment]
Merci bcq pour la correction. Vous voulez une autre trouvaille ? celle là elle est assez fourbe de la part de Parionsport. Et ne vous inquiétez pas si c trop galère à corriger je supprime le match dans l'interface. Cela concerne le match de FA CUP le 10 février entre Sheffield et Bristol : 
![image](https://user-images.githubusercontent.com/78261069/106938563-6f118b80-671f-11eb-8718-10dd8e410c97.png)

Tout laisse croire effectivement sur l'image suivante que Sheffield est à 5.50 : 
![image](https://user-images.githubusercontent.com/78261069/106938656-923c3b00-671f-11eb-87e2-44a0a4b6c471.png)

et du coup le bot aussi pense ça...

Mais PSEL incite à l'erreur car c'est Sheffield -2 !! à 5.50 ! 

C assez honteux de leur part je trouve : 

![image](https://user-images.githubusercontent.com/78261069/106938793-b8fa7180-671f-11eb-9759-0f843d8332f5.png)

**Reply by pretrehr** [reply]
Ah oui celui-là est très fourbe ahah
Après c'est toujours possible d'au moins éviter l'affichage de ce genre d'erreurs en vérifiant que pour chaque match récupéré, il n'y a pas de surebet trop gros. Je vais y réfléchir

**Comment by kreon76** [comment]
Je vais supprimer le match jusqu au 10 février ça me dérange pas...
